tests/unit/sagemaker/jumpstart/test_notebook_utils.py

```diff
@@ -574,10 +575,6 @@ def vulnerable_training_model_spec(bucket, key, *args, **kwargs):
 
         assert patched_read_s3_file.call_count == 0
 
-    @pytest.mark.skipif(
```

Comment:
i'm still confused why this was flakey in the first place and how your PR fixed this

Reply:
I think this was flaky before we introduced prop models, but somehow seen failing more often after prop models. I updated to only list open weight models, and tried starting the test in multiple threads in parallel locally and they haven't been failing once for me. I think there is less parallelism in PR hooks now, so hope this is not flaky anymore.